internalstaging.theanvil.io/manifest.json

```diff
@@ -57,7 +57,7 @@
       "action": "export",
       "container": {
         "name": "job-task",
-        "image": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/pelican-export:0.5.1",
```

Comment:
pelican-export 0.5.1 is NOT in 2020.10, but this is ok because we only need the 0.5.1 image if we're doing PFB exports from the Files tab.

Reply:
THANKS A MILLION for checking @mpingram .
Quick question, do I need to add the pelican-export-files to this manifest? 🤔